server/src/binary/mapper.rs

```diff
@@ -268,5 +262,12 @@ fn extend_user(user: &User, bytes: &mut BytesMut) {
 fn extend_pat(personal_access_token: &PersonalAccessToken, bytes: &mut BytesMut) {
     bytes.put_u8(personal_access_token.name.len() as u8);
     bytes.put_slice(personal_access_token.name.as_bytes());
-    bytes.put_u64_le(personal_access_token.expiry.unwrap_or(0));
+    match &personal_access_token.expiry_at {
```

Comment:
what about implementing as_bytes to expiry at?

Reply:
Internally it's just u64, and the match exists due to this field being an optional type.